Dockerfile

```diff
@@ -93,45 +93,6 @@ RUN \
     rm -rf /usr/local/src/steem/build ; \
     fi
 
-RUN \
```

Comment:
We can't get rid of this build. This is the build that makes the testnet binary.